src/component/elements/formik/FormikCheckBox.tsx

```diff
@@ -1,18 +1,27 @@
 import { useFormikContext } from 'formik';
 import lodashGet from 'lodash/get';
-import PropTypes from 'prop-types';
 import { useCallback } from 'react';
 
-import CheckBox from '../CheckBox';
+import CheckBox, { CheckBoxProps } from '../CheckBox';
+
+interface FormikCheckBoxProps extends CheckBoxProps {
+  name: string;
+  label?: string;
+  className?: string;
+  onChange?: (element: any) => void;
+  reverse?: boolean;
+}
+
+function FormikCheckBox(props: FormikCheckBoxProps) {
+  const {
+    label,
+    name,
+    onChange = () => null,
+    className = '',
```

Comment:
```suggestion
    className = 'checkbox',
```

Maybe keep this default? I'm not sure if it's used.